ansible/group_vars/all/config.yml

```
@@ -1049,6 +1049,11 @@ configuration_all:
     server: true
     client: false
     nuxtclient: false
+  BOARD_COLLABORATION_URI:
+    value: ""
```

Only add an default value if you want to set them to an default value who is not empty and "" is to see as empty
mabey set the websocet to the domain is an good idee
```suggestion
    value: "ws://{{ DOMAIN }}"
```

---

fixed as suggested